Updated a known issue reported by customers (#5396)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/information-barriers-in-teams.md

For more details, including plans and pricing, see [Licensing Guidance](https://docs.microsoft.com/office365/servicedescriptions/microsoft-365-service-descriptions/microsoft-365-tenantlevel-services-licensing-guidance/microsoft-365-security-compliance-licensing-guidance).

## Known Issues

> - **Users cant join meetings** : If IB policies are enabled, users are not allowed to join meetings IF meeting roster size is more than 250 users. Root cause is that we IB checks rely on whether users can be added to meeting chat roster and take that signal to allow user to join meeting. Joining a meeting once will add that user to roster, hence for recurring meetings, the roster fills up fast. Once it reaches to a count of 250 users, no additional users are allowed to be added to meeting chat roster. If IB is enables, users are not allowed to join meeting, but if IB is not enabled, users are allowed to join meeting, though they wont be added to meeting chat roster. Short term solution is to remove inactive members from meeting chat roster to make space for new users. Later, we are going to increase roster size of meeting chat and will fix engg code.

Reviewer: ```suggestion
- **Users can't join meetings**: If IB policies are enabled, users are not allowed to join meetings IF the meeting roster size is more than 250 users. The root cause is that IB checks rely on whether users can be added to a meeting chat roster and takes that signal to allow users to join meetings. Joining a meeting once will add that user to the roster, hence for recurring meetings, the roster fills up fast. Once it reaches a count of 250 users, no additional users are allowed to be added to the meeting chat roster. If IB is enabled, users are not allowed to join the meeting, but if IB is not enabled, users are allowed to join the meeting, though they won't be added to the meeting chat roster. A short term solution is to remove inactive members from the meeting chat roster to make space for new users. The size of meeting chat rosters will be increased at a later date and the engg code will be fixed.
```
